raiden-cli/src/utils/channels.ts

```diff
@@ -27,18 +27,16 @@ export function filterChannels(
 }
 
 /* eslint-disable @typescript-eslint/camelcase */
-export function transformSdkChannelFormatToApi(channels: RaidenChannel[]): ApiChannel[] {
-  return channels.map((channel) => {
-    return {
-      channel_identifier: channel.id ?? 0, // FIXME: old "bug" in the SDK
-      token_network_address: channel.tokenNetwork,
-      partner_address: channel.partner,
-      token_address: channel.token,
-      balance: channel.balance.toString(),
-      total_deposit: channel.ownDeposit.toString(),
-      state: channel.state,
-      settle_timeout: channel.settleTimeout ?? 0,
-      reveal_timeout: 0, // FIXME: Not defined here. Python client handles reveal timeout differently,
-    } as ApiChannel;
-  });
+export function transformSdkChannelFormatToApi(channel: RaidenChannel): ApiChannel {
+  return {
+    channel_identifier: channel.id ?? 0, // FIXME: old "bug" in the SDK
+    token_network_address: channel.tokenNetwork,
+    partner_address: channel.partner,
+    token_address: channel.token,
+    balance: channel.balance.toString(),
+    total_deposit: channel.ownDeposit.toString(),
+    state: channel.state,
+    settle_timeout: channel.settleTimeout ?? 0,
+    reveal_timeout: 0, // FIXME: Not defined here. Python client handles reveal timeout differently,
+  } as ApiChannel;
```

Comment:
Nitpick: don't type-assert here, this could hide issues if something is wrong with your object-literal declaration. Instead, have the type declared on an variable assignment or return type (as you did) and return/assign the object-literal directly, this way TSC will complain if something isn't right, and you can fix it explicitly.

Reply:
:heavy_check_mark: